visualization/app/codeCharta/ui/fileChooser/fileChooser.e2e.ts

```diff
@@ -27,9 +30,14 @@ describe("FileChooser", () => {
 		expect(await page.$eval("#loading-gif-map", el => el["className"])).toContain("ng-hide")
 	})
 
-	it("should not load non json file", async () => {
+	it("should open an invalid file, close the dialog and open a valid file", async () => {
 		await fileChooser.openFile("./app/codeCharta/assets/logo.png")
+		expect(await dialogError.getMessage()).toEqual(" file is empty or invalid")
 
-		expect(await filePanel.getSelectedName()).toEqual("sample1.cc.json")
+		await dialogError.clickOk()
+
+		await FileChooserPageObject.selectFile("./app/codeCharta/assets/sample3.cc.json")
+
+		expect(await filePanel.getSelectedName()).toEqual("sample3.cc.json")
 	})
```

Comment:
Please add another test that checks for two files. The first is valid but contains warnings, the second is invalid. This will likely still cause issues.

Reply:
Alright, added an e2e test. The warning message is closing automatically and then the error message appears, which needs user interaction. Error dialogs are not being closed if a warning message should follow.